packages/opentelemetry-resources/src/ResourceImpl.ts

```diff
@@ -177,3 +188,24 @@ function guardedRawAttributes(
     return [k, v];
   });
 }
+
+function mergeSchemaUrl(
+  base: Resource,
+  other: Resource | null
```

Comment:
nit: using the same names as in the spec (`old` and `updating`) can help make it easier to cross check between implementation and specification.

https://opentelemetry.io/docs/specs/otel/resource/sdk/#merge